.wiby.json

```
@@ -9,5 +9,6 @@
     {
       "repository": "git+https://github.com/wiby-test/pass"
     }
-  ]
+  ],
+  "pullrequest": false
```

I think this should be an option for each dependent. This does not depend on `wiby` user, but on the configuration of CI in the target repo.

---

```suggestion
  "pullRequest": false
```

And `--pull-request` for params in `commands`?

Prior-art: `dry-run` in `wiby clean` and also Yargs will automagically convert the `-` to `camelCase` too.

---

Just figured this might be ambiguous, so...

1. If we're calling `wiby test --dependent [dependent url]`, then we're only running for a single dependent, and `--pull-request` applies to just that - already implied and works
2. If we're calling `wiby test --pull-request` for _all_ dependents, then the CLI option needs to override any configuration options, i.e. it should open pull requests for all dependents defined in `wiby.json`
3. If we're calling `wiby test` without `--pull-request`, then it should only open pull requests for dependents that have `pullRequest: true` configured.

Would that make sense?

---

yes, this makes sense.